### PR TITLE
Prescription Security glasses

### PR DESCRIPTION
**Pull request by SomeCabbage**
The old PR broke so I redid it.

# Description

Adds prescription security glasses
they have a small chance to fall off your face when you slip. Small enough that it wont be a problem normally but just enough so that it'll be noticeable

Adds 2 units to SecTech
Adds to loadouts for 4 points

# Media
image

### CODE REVIEW

File: Resources/Prototypes/SimpleStation14/Entities/Clothing/Eyes/glasses.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - type: Clothing
+    sprite: Clothing/Eyes/Glasses/secglasses.rsi
+  - type: VisionCorrection
+  - type: ClothingGrantTag
+    tag: GlassesNearsight
+  - type: FlashImmunity
+  - type: EyeProtection
+    protectionTime: 5
+  - type: DropOnSlip
+    chance: 5
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Clothing/Eyes/glasses.yml:11]:
```suggestion
    chance: 5
    
```

File: Resources/Prototypes/SimpleStation14/Entities/Clothing/Eyes/glasses.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+- type: entity
+  parent: ClothingEyesBase
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Clothing/Eyes/glasses.yml]: Parent from Secglasses, and remove the redundant components. Just add the prescription glasses thing and the drop chance.

File: Resources/Prototypes/SimpleStation14/Loadouts/traits.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Loadouts/traits.yml:1]: Make sure this is in the same order as the other loadouts. I believe I ordered them something like Department alphabetically? Not certain, just try to find the pattern and match it.

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/sec.yml [hunk 3]
```diff
@@ -7,6 +7,7 @@
     Flash: 5
     FlashlightSeclite: 5
     ClothingEyesGlassesSecurity: 4
+    ClothingEyesGlassesSecurityPrescription: 2
```
Comment (Pspritechologist) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/sec.yml]: Move this to the bottom. Just for the sake of organization (and merge conflicts), it's nice to have anything we add at the bottom of a list.

**Review by Pspritechologist — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/SimpleStation14/Loadouts/traits.yml
Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Loadouts/traits.yml:1]: you mean the whitelisted jobs list?

Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Loadouts/traits.yml:1]: for other items it seems to be based off the "rank" of the position or for epistemics items its just a list of the jobs

Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Loadouts/traits.yml:1]: I just did it in alphabetical order this time

**Review by Pspritechologist — APPROVED** [review]

**Comment by Pspritechologist** [comment]
Update the pr description before merge.
It probably doesn't need a changelog.

**Comment by SomeCabbage** [comment]
no more changelog

File: Resources/Prototypes/SimpleStation14/Loadouts/traits.yml [hunk 4]
```diff
@@ -24,3 +24,19 @@
...
+  category: Traits
+  cost: 4
+  jobWhitelist:
+    - HeadOfSecurity
+    - SecurityCadet
+    - SecurityOfficer
+    - Warden
+  whitelist:
+    components:
+      - Nearsighted
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Loadouts/traits.yml:41]: I wonder if this works as intended (if it doesn't I'm going to explode, hold on)

Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/SimpleStation14/Loadouts/traits.yml:41]: alright I am good coder man

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Pull request merged**